docs/source/about/guides/linux.rst

```diff
@@ -0,0 +1,459 @@
+
+	An incorrect configuration will prevent the sshd service from starting, which might mean losing access to reach the server.
+
+.. tab:: SystemD
+
+    .. code-block:: bash
+
+		sudo systemctl restart sshd.service
+
+.. tab:: Runit
+
+    .. code-block:: bash
+
+		sudo sv restart sshd
+
+.. tab:: OpenRC
+
+    .. code-block:: bash
+
+		sudo rc-service sshd restart
+
+----
+
+Virtual Display Setup
++++++++++++++++++++++
+
+There is no need to buy a dummy plugin, when you can just use this config to get a virtual display for free!
+
+.. important:: This is only available for NVidia GPUs using Xorg
+
+.. code-block::  
+
+	Section "ServerLayout"
+		Identifier     "TwinLayout"
+		Screen         0 "metaScreen" 0 0
+	EndSection
+
+	Section "Monitor"
+		Identifier     "Monitor0"
+		Option         "Enable" "true"
+	EndSection
+
+	Section "Device"
+		Identifier     "Card0"
+		Driver         "nvidia"
+		VendorName     "NVIDIA Corporation"
+		Option         "MetaModes"          "1920x1080"
+		Option         "ConnectedMonitor"   "DP-0"
+		Option         "ModeValidation" "NoDFPNativeResolutionCheck,NoVirtualSizeCheck,NoMaxPClkCheck,NoHorizSyncCheck,NoVertRefreshCheck,NoWidthAlignmentCheck"
+	EndSection
+
+	Section "Screen"
+		Identifier     "metaScreen"
+		Device         "Card0"
+		Monitor        "Monitor0"
+		DefaultDepth    24
+		Option         "TwinView" "True"
+		SubSection "Display"
+			Modes          "1920x1080"
+		EndSubSection
+	EndSection
+
+.. note::
+	The ``ConnectedMonitor`` tricks the GPU into thinking a monitor is connected, even if there is none actually connected! 
+	This allows a virtual display to be created that is accelerated with your GPU! The ``ModeValidation`` option disables valid resolution checks,
+	so you can choose any resolution on the host!
+	
+	**References**
+
+	* `issue comment on virtual-display-linux <https://github.com/dianariyanto/virtual-display-linux/issues/9#issuecomment-786389065>`_
+	* `Nvidia Documentation on Configuring TwinView <https://download.nvidia.com/XFree86/Linux-x86/270.29/README/configtwinview.html>`_
+	* `Arch Wiki Nvidia#TwinView <https://wiki.archlinux.org/title/NVIDIA#TwinView>`_
+	* `Unix Stack Exchange - How to add virtual display monitor with Nvidia proprietary driver <https://unix.stackexchange.com/questions/559918/how-to-add-virtual-monitor-with-nvidia-proprietary-driver>`_
+
+
+Uinput Permissions Workaround
++++++++++++++++++++++++++++++
+
+**Steps**
+
+We can use ``chown`` to change the permissions from a script. Since this requires ``sudo``, we will need to update the sudo configuration to execute this without being prompted for a password.
+
+#. Create a ``sunshine-setup.sh`` script to update permissions on ``/dev/uinput``. Since we aren't logged into the host, the udev rule doesn't apply.
+#. Update user sudo configuration ``/etc/sudoers.d/<user>`` to allow the ``sunshine-setup.sh`` script to be executed with ``sudo``.
+
+.. admonition:: Why is this necessary?
```

Comment:
Sorry, I went trigger happy with them lol. I was excited about the color boxes since plain markdown doesnt have any sort of coloring. I agree with you, I think having consistent admonitions is better.

Reply:
They have actually added this to GitHub flavored markdown, but not too extensively. I don't know if it's documented anywhere, but these are two that I've discovered so far.

> **Note**
> My awesome note!


> **Warning**
> You have been warned!

```md
> **Note**
> My awesome note!
```